### ISSUE TITLE
client returning UnknownBeatmap on converts

### ISSUE THREAD

**Issue by de-odex** [llllllllll/slider#14]
```
Traceback (most recent call last):
  File "C:\Users\deodex\AppData\Local\Programs\Python\Python36\lib\site-packages\slider\client.py", line 647, in beatmap
    converted, = converted
ValueError: not enough values to unpack (expected 1, got 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "bot.py", line 183, in sendpp
    beatmap_data_api = self.osu_api_client.beatmap(beatmap_id=beatmap_id, include_converted_beatmaps=True, game_mode=slider.game_mode.GameMode(mode))
  File "C:\Users\deodex\AppData\Local\Programs\Python\Python36\lib\site-packages\slider\client.py", line 656, in beatmap
    raise UnknownBeatmap(kind, id_)
slider.client.UnknownBeatmap: no beatmap found that matched id: <built-in function id>
```
with 
`beatmap_data_api = self.osu_api_client.beatmap(beatmap_id=beatmap_id, include_converted_beatmaps=True, game_mode=slider.game_mode.GameMode(mode))`
where 
```
self.osu_api_client = slider.client.Client(osu_library, config.api_key)
beatmap_id = "818062"  # example
mode = 2
```

**Comment by de-odex** [comment]
***Fix***
`include_converted_beatmaps` must equal `1` for this to work, not `True`.

**Comment by llllllllll** [comment]
Thanks, I can normalize the passed value to an integer before making the request.

**Comment by llllllllll** [comment]
fixed in #15
